pdns/syncres.hh

```diff
@@ -697,6 +697,7 @@ public:
 
   std::unordered_map<std::string,bool> d_discardedPolicies;
   DNSFilterEngine::Policy d_appliedPolicy;
+  unsigned int d_authzonequeries;
```

Comment:
This variable should be initialized [here](https://github.com/rgacogne/pdns/blob/f7b8cffae1ff94c2e78ac6f3b3ee494735a0c23c/pdns/syncres.cc#L111)